### PR TITLE
Add enterprise course enrollments data

### PR DESCRIPTION
**Pull request by long74100**
- Add enterprise course enrollments data to co

Open to other suggestions of where to put this data. It made sense to put it under course enrollments for easy access.

https://openedx.atlassian.net/browse/ENT-5253

Screen Shot 2022-01-24 at 2 09 37 PM
urse enrollments

### CODE REVIEW

**Review by macdiesel — APPROVED** [review]

**Review by manny-m — APPROVED** [review]
LGTM 👍

File: src/users/data/test/enrollments.js [hunk 1]
```diff
@@ -43,6 +43,19 @@ export const enrollmentsData = [
...
+    enterpriseCourseEnrollments: [
+      {
+        course_id: 'course-v1:testX+test123+2030',
+        enterprise_customer_name: 'Test Enterprise',
+        enterprise_customer_user_id: 1,
+        license: {
+          uuid: 'fake-license-uuid',
+          is_revoked: false,
+        },
+        saved_for_later: false,
```
Comment (Ali-D-Akbar) [inline src/users/data/test/enrollments.js]: `saved_for_later: false` is never used in the component so perhaps this can be removed for the test.

**Review by Ali-D-Akbar — APPROVED** [review]
One nit although not a blocker. 👍

File: src/users/enrollments/v2/Enrollments.jsx [hunk 2]
```diff
@@ -201,17 +208,50 @@ export default function Enrollments({
...
-        data={[row.original.expander]}
-        columns={extraColumns}
-        styleName="custom-expander-table"
-      />
+      <>
+        <TableV2
+          data={[row.original.expander]}
+          columns={extraColumns}
+          styleName="custom-expander-table"
+        />
```
Comment (DawoudSheraz) [inline src/users/enrollments/v2/Enrollments.jsx:242]: You can add a divider between these two tables for better readability.

File: src/users/enrollments/v2/Enrollments.jsx [hunk 3]
```diff
@@ -201,17 +208,50 @@ export default function Enrollments({
...
+                  accessor: 'consentRequired',
+                },
+                {
+                  Header: 'License',
+                  accessor: 'license',
+                },
+                {
+                  Header: 'License Revoked',
+                  accessor: 'isLicenseRevoked',
+                },
```
Comment (DawoudSheraz) [inline src/users/enrollments/v2/Enrollments.jsx]: For the sake of consistency, please define these column values in a variable and use the variable here.

File: src/users/enrollments/v2/Enrollments.test.jsx [hunk 4]
```diff
@@ -88,20 +88,33 @@ describe('Course Enrollments V2 Listing', () => {
     expect(wrapper.find('changeEnrollmentForm')).toEqual({});
   });
 
-  it('Enrollment extra data is rendered for individual enrollment', () => {
+  it('Enrollment extra data and enterprise course enrollments are rendered for individual enrollment', () => {
```
Comment (DawoudSheraz) [inline src/users/enrollments/v2/Enrollments.test.jsx:91]: please add a unit test that checks Table is not rendered for Enterprise course enrollments if the data is not present in the API.

**Review by DawoudSheraz — CHANGES_REQUESTED** [review]
Small updates and a unit test case to add. Otherwise, the changes are good.

**Review by DawoudSheraz — APPROVED** [review]
Good to merge once the build is green.

**Pull request merged**